kinto/config-check-test/test_kinto_signer.py

```diff
@@ -35,29 +35,26 @@ def verify_signatures(client):
         return 0
 
 
+def get_signer_id(client):
+    dest_col = client.get_collection()
```

Comment:
Rather than getting the collection data twice (once here and once in `verify_signature`, what about doing something like that:

```
def get_collection_data(client):
     collection = client.get_collection()
     records = client.get_records(_sort='-last_modified')
     return collection, records

def verify_signer_id(collection, key_name):
    assert collection['data']['signature']['signer_id'] == key_name
```

And then calling:
```
 def test_addons_signatures(env, conf):
     client = Client(
         server_url=conf.get(env, 'server'),
         bucket='blocklists',
         collection='addons'
     )
    collection, records = get_collection_data(client)
    try:
        assert verify_signatures(collection, records)
        assert verify_signer_id(collection, "onecrl_key")
    except KintoException as e:
        if e.response.status_code == 401:
             pytest.fail('blocklists/addons does not exist')
        pytest.fail('something went wrong: %s %s" % (e.response.status_code, e.response))
```